src/types/dto/review.ts

```diff
@@ -29,7 +35,15 @@ export interface EditedMediaDto extends BaseEditedItemDto {
   type: "file" | "image"
 }
 
-export type EditedItemDto = EditedPageDto | EditedConfigDto | EditedMediaDto
+export interface EditedPlaceholderDto extends BaseEditedItemDto {
+  type: "placeholder"
+}
+
+export type EditedItemDto =
```

Comment:
if we do want to return this to the frontend, we probably need an associated PR on the frontend that either hides the placeholder files or does something useful with them.

i'm of hte mind to filter them uot atm